detection_2d/BoundingBox3D[] result
